### PR TITLE
Add dynamically generated BNLS server list

### PR DESCRIPTION
**Pull request by carlbennett**

### CODE REVIEW

File: web/bnls.php [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+
+if (!($data = file_get_contents($url))) {
+  exit_here(sprintf('Failed to download server list from: %s', $url));
+}
+
+if (!($json = json_decode($data, true))) {
+  exit_here(sprintf('JSON error #%d while decoding list', json_last_error()));
+}
+
+echo "\n"; // not sure why this is here - @carlbennett
```
Comment (carlbennett) [inline web/bnls.php:20]: Not sure why this extra blank line is here. I duplicated it from the static code this script is replacing (see https://github.com/stealthbot/StealthBot/pull/83/files#diff-7d67a09e3b12d93e23a7d59e416c4e1aL3)

Comment (Davnit) [reply web/bnls.php:20]: I think it's used as a check for good data return.
https://github.com/stealthbot/StealthBot/blob/master/trunk/frmChat.frm#L2359-L2393

Comment (Davnit) [reply web/bnls.php:20]: Or not... dunno.

Comment (carlbennett) [reply web/bnls.php:20]: I believe StealthBot uses this in its UI to de-select a BNLS server in the combobox/dropdown menu field. The first entry needs to be blank for this purpose.

**Pull request merged**